packages/alfa-performance/src/performance.ts

```diff
@@ -0,0 +1,211 @@
+import { Callback } from "@siteimprove/alfa-callback";
+import { Emitter } from "@siteimprove/alfa-emitter";
+import { Serializable } from "@siteimprove/alfa-json";
+
+import * as json from "@siteimprove/alfa-json";
+
+import { now } from "./now";
+
+export class Performance<T>
+  implements
+    AsyncIterable<Performance.Entry<T>>,
+    Serializable<Performance.JSON> {
+  public static of<T>(): Performance<T> {
```

Comment:
Should `T` default to `string` which sounds like it's going to be a frequent case?
(would simplify calls by removing need to provide the type each time)